frontend/src/apps/main/entry/index.tsx

```diff
@@ -8,19 +8,24 @@ import { Footer } from "t9/apps/main/components/footer";
 import { mainStore } from "t9/apps/main/redux";
 import { Provider } from "react-redux";
 
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
 const Landing = lazy(() => import("t9/apps/main/scenes/landing"));
-const Articles = lazy(() => import("t9/apps/main/scenes/articles"));
+const Articles = lazy(() => import("t9/apps/main/scenes/Articles"));
```

Comment:
```suggestion
const Articles = lazy(() => import("t9/apps/main/scenes/articles"));
```
all filenames should be python cased, please rename the file too